package.json

```diff
@@ -154,9 +154,9 @@
     "detect-indent": "6.1.0",
     "editorconfig": "0.15.3",
     "email-addresses": "5.0.0",
-    "emoji-regex": "9.2.2",
-    "emojibase": "5.2.0",
-    "emojibase-regex": "5.1.3",
+    "emoji-regex": "1.0.0",
```

Comment:
Are you sure about the downgrade  from v9 to v1 ?

Reply:
Gosh 🤦‍♂️ Thank you